tests/FloatWithinTests.elm

```diff
@@ -35,6 +35,16 @@ floatWithinTests =
                         (radius * pi)
                             |> Expect.within (Relative 0.0001) (radius * 3.14)
             ]
+        , describe "negative use-cases"
```

Comment:
Is the use-case negative? How about `negative arguments`?

Reply:
We don't want a negative tolerance, which is one of the arguments, so I went with "use-cases with negative nominal and/or actual values"